src/modules/data/blocks/ticket/selectors.js

```diff
@@ -268,7 +268,7 @@ export const getTicketHasBeenCreated = createSelector(
 
 export const getTicketHasChanges = createSelector(
 	[ getTicket ],
-	( ticket ) => ticket.hasChanges,
+	( ticket ) => ticket.hasChanges || ! ticket.ticketId,
```

Comment:
Adding this `! ticket.ticketId` will trigger change detection on each change of the ticket during its creation, is this intentional? How does this work when the SLR feature is disabled?

Reply:
Couldn't see any change ... reverted since it doesn't cause any issues.